src/server/config.ts

```diff
@@ -450,6 +450,8 @@ export const BffEndpoints = {
   // AFIS
   AFIS_BUSINESSPARTNER:
     '/services/afis/businesspartner/:businessPartnerIdEncrypted',
+  AFIS_FACTUREN: '/services/afis/facturen/:state/:businessPartnerIdEncrypted',
```

Comment:
Volgens mij kun je hier ook al de waardes van status definiëren.
```
  AFIS_FACTUREN: '/services/afis/facturen/(open|closed)/:businessPartnerIdEncrypted',
```

Reply:
Zo wordt deze param gechecked door het framework. Als 't goed is zou een andere waarde dan open en closed een error moeten geven.